Create NI-SWITCH message implementation

rtzoeller opened the pull request:
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-slsc-switch-message-library/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Create an NI-SWITCH variant of the SLSC Switch message library.

### Why should this Pull Request be merged?

We want to allow using NI-SWITCH hardware with the Routing and Faulting custom device.

### What testing has been done?

Tested changes in conjunction with https://github.com/ni/niveristand-routing-and-faulting-custom-device/pull/220.

buckd reviewed (commented):
- [ ] Do we need the SET caps file for NI-SWITCH tests?

The tests hang on consumer initialization in `setUp.vi` on my machine. Is there a specific version of the base messaging library I need?

rtzoeller commented:
> Do we need the SET caps file for NI-SWITCH tests?

No, good catch.

> The tests hang on consumer initialization in setUp.vi on my machine in 2017. Is there a specific version of the base messaging library I need?

There shouldn't be. I'll reach out directly.

rtzoeller commented:
@buckd I've added the same workaround as is used in the custom device, which seems to prevent the hang. I've also removed the extraneous test asset.
